LedgerHQ/ledger-live-desktop
# LL-8212 Add missing animation for fw update LNX

## 🦒 Context (issues, jira)
https://ledgerhq.atlassian.net/browse/LL-8212


## 💻  Description / Demo (image or video)
We were missing the asset for firmware updating a nano X, for some reason it was hardcoded that it would use the asset for a nano S instead. After getting the right animation from our wonderful designers, this pr introduces the asset into its rightful place.
<img width="490" alt="image" src="https://user-images.githubusercontent.com/4631227/148548255-035e75fb-0c2b-4c46-bfc4-c6129a1fb346.png">



## 🖤  Test plan
- Launch app with the `DEBUG_THEME` flag
- Start a firmware update flow with a Nano X
- Ensure the assets are for a Nano X and not a Nano S
- When in the progress step, use the theme helper to toggle between themes
- Profit

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [x] a specific test planned is defined here
- **on delivery**: at least one of these two checkboxes must be checked: 

PR must pass CI, merge develop if conflicts, do not force push. Thanks!
